storybook/stories/Buttons/Button/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '../../../../src/components/Button'
+
+export default ButtonStory = ({ label, icon, isDark, isDisabled, handlePress, iconPosition }) => {
+  return (
+    <Button label={'Test Label'} icon={icon} isDark={isDark} isDisabled={isDisabled} onPress={handlePress} iconPosition={iconPosition}/>
```

Comment:
If a component has more then two props, I would recommend to write it in multiple rows:
```
<Button
  label={'Test Label'}
  icon={icon}
  isDark={isDark}
  isDisabled={isDisabled}
  onPress={handlePress}
  iconPosition={iconPosition}
/>
```